.pre-commit-config.yaml

```diff
@@ -35,8 +35,8 @@ repos:
       name: pprettier
       language: system
       require_serial: true
-      entry: "bash -c 'yarn install --silent && yarn run pprettier --write \"$@\"'"
-      types: [ts]
+      entry: "bash -c 'yarn install --silent && yarn run pprettier --write \"$@\"' bash"
```

Comment:
What is that `bash` at the end?

Reply:
That's just an argument that ends up in `$0`. Could be anything really, but since `$0` is usually the name of the program I choose that. Before, the first file name ended up in `$0` and that's why it was skipped.

Reply:
Maybe worth adding a comment about it. 🙂

Reply:
Done.